app/controllers/topics_controller.rb

```diff
@@ -11,7 +12,8 @@ def new
   end
 
   def create
-    @topic = Topic.create(topic_params)
+    @topic = Topic.new(topic_params)
+    @topic.user_id = current_user.id
```

Comment:
👍 That works, but the following is better (and is a one liner). It uses the fact that rails allows you to create associations via the main object, and this way automatically sets the foreign key (i.e. user_id) for you.
```ruby
@topic = current_user.topics.create(topic_params)
```